plugins/woocommerce/tests/e2e-pw/tests/shopper/cart-block-calculate-shipping.spec.js

```diff
@@ -202,12 +220,21 @@ test.describe( 'Cart Block Calculate Shipping', () => {
 		await addAProductToCart( page, product1Id );
 		await page.goto( cartBlockPage.slug );
 
+		await page
+			.locator( 'button', {
+				hasText: 'Enter the address to check delivery options',
+			} )
+			.click();
```

Comment:
See my previous comment regarding using `page.getByRole()` instead of `page.locator()`.